modules/internal/ChapelDynamicLoading.chpl

```diff
@@ -204,7 +205,7 @@ module ChapelDynamicLoading {
       var ret: unmanaged chpl_BinaryInfo? = nil;
 
       on Locales[0] {
-        ref store = chpl_binaryInfoStore;
+        const store = chpl_binaryInfoStore.borrow();
         var shouldCreateNewEntry = true;
```

Comment:
Why do this at all? Can't you just remove the definition of `store` and use `chpl_binaryInfoStore` directly